### ISSUE TITLE
xanthoastrocytoma

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6763]
**Preferred term label:**
xanthoastrocytoma

**Synonyms**

**Definition (free text, please give PubMed ID)**
A rare astrocytoma (WHO grade II or III), often a well-circumscribed cystic component and presenting with seizures.

30051528
12790896

**Parent term (use hpo.jax.org/app)**
Astrocytoma HP:0009592

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Multiple reports in NF1
OMIM: 162200

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
We should try to use the terminology of NCIT
What is the relationship to this hierarchy?
https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C4323
@DLewisSmith

**Comment by DLewisSmith** [comment]
Then I would suggest renaming and coreferencing to the concept you link to:
Pleomorphic xanthoastrocytoma (http://purl.obolibrary.org/obo/NCIT_C4323) is a Astrocytoma HP:0009592.

**Comment by pnrobinson** [comment]
added, thanks
